modules/denoise/denoise_wrapper.cpp

```diff
@@ -29,7 +29,9 @@
 /**************************************************************************/
 
 #include "denoise_wrapper.h"
-#include "thirdparty/oidn/include/OpenImageDenoise/oidn.h"
+
+#include <OpenImageDenoise/oidn.h>
```

Comment:
Is this actually cleaner?

Reply:
Yes, this would be a necessary step to compile against a system provided oidn library on Linux:
```
$ urpmq -l lib64openimagedenoise-devel
/usr/include/OpenImageDenoise
/usr/include/OpenImageDenoise/config.h
/usr/include/OpenImageDenoise/oidn.h
/usr/include/OpenImageDenoise/oidn.hpp
/usr/lib64/cmake/OpenImageDenoise-1.4.3
/usr/lib64/cmake/OpenImageDenoise-1.4.3/OpenImageDenoiseConfig-relwithdebinfo.cmake
/usr/lib64/cmake/OpenImageDenoise-1.4.3/OpenImageDenoiseConfig.cmake
/usr/lib64/cmake/OpenImageDenoise-1.4.3/OpenImageDenoiseConfigVersion.cmake
/usr/lib64/libOpenImageDenoise.so
```

Thirdparty includes should ideally be added with "system-like" paths which match either their base location from pkgconfig (here `/usr/include`) or from our custom added include path (`#thirdparty/oidn/include`).